docker/py3-native/README.md

```diff
@@ -3,7 +3,71 @@
+
+
+## How To Handle Native Images
+### What is a native image?
+* Native image is a combination of a lot of docker images that scripts and integrations use. 
+* The native image contains all the OS / python dependancies that are used in the supported docker images
+* For example, given the docker image taxii2, tesseract and chromium, the native image will contain all of their OS / python dependencies. 
+* That means that every integration/script that uses taxii2, tesseract or chromium docker image can also be used in the native image and there is compatibility between them.
+
+### Required: What should I do when changing a docker-image that is supported in the native image?
+1) Check if the docker image that was changed is supported also in the native image. It is possible to check it in the [docker native image configuration file](https://github.com/demisto/content/blob/master/Tests/docker_native_image_config.json). **Note:** if you have changed a docker-image that is supported in the native image the **native docker validator workflow** will fail and alert you. 
+2) If the docker image is supported by the native image, apply the same changes to the native image by the following scenarios:
+   - If a new python dependency was added to the docker image, make sure it's also added to the native image, examples:  
+      - assuming "pan-os-python" python module was added to the **[pan-os-python](https://github.com/demisto/dockerfiles/tree/master/docker/pan-os-python)** docker image, make sure to add the "pan-os-python" module library also to the native image.
+      - assuming "beautifulsoup4" python module was added to the **[py3ews](https://github.com/demisto/dockerfiles/tree/master/docker/py3ews)** docker image, make sure to add the "beautifulsoup4" python module also to the native image.
+   - If a new OS dependency was added to the docker image, make sure it's also added to the native image and also documented in [here](#os-dependencies-for-each-supported-docker-image) readme, examples:
+      - assuming "git" was added to the **[crypto](https://github.com/demisto/dockerfiles/tree/master/docker/crypto)** docker image, make sure it is also added to the native image and make sure its documented [here](#os-dependencies-for-each-supported-docker-image).
+      - assuming "curl" was added to the **[readpdf](https://github.com/demisto/dockerfiles/tree/master/docker/readpdf)** docker image, make sure it is also added to the native image and make sure its documented [here](#os-dependencies-for-each-supported-docker-image).
+3) After you are done, add to your PR the label "native image approved" that means that the native image is compatible with the updated docker image that you changed.
+4) **Add the script/integration to be ignored only in the production native images, refer the section [How to ignore native-images in the docker native image configuration file?](#how-to-ignore-native-images-in-the-docker-native-image-configuration-file).**
```

Comment:
```suggestion
1) Check if the docker image that was changed is supported also in the native image. That can be checked in the [docker native image configuration file](https://github.com/demisto/content/blob/master/Tests/docker_native_image_config.json). **Note:** if you have changed a docker-image that is supported in the native image the **native docker validator workflow** should fail and alert you. 
2) If the docker image is supported by the native image, make any necessary changes to the native image to ensure it can still run the integration/script.
- If the changes cant be made and the script can no longer run with the native image, have the native image ignore this script in the config file. 
- Even if the same changes can be made, it's likely that we will need to drop support in the current version of the native image since that image doesnt have these changes. If this is the case we should still add this script to the ignore list (for the current version, not dev version).


3) After you are done, the PR reviewer should add the label "native image approved" that means that the native image is compatible with the updated docker image that you changed.

Reply:
the following :

  - If a new python dependency was added to the docker image, make sure it's also added to the native image, examples:  
      - assuming "pan-os-python" python module was added to the **[pan-os-python](https://github.com/demisto/dockerfiles/tree/master/docker/pan-os-python)** docker image, make sure to add the "pan-os-python" module library also to the native image.
      - assuming "beautifulsoup4" python module was added to the **[py3ews](https://github.com/demisto/dockerfiles/tree/master/docker/py3ews)** docker image, make sure to add the "beautifulsoup4" python module also to the native image.
   - If a new OS dependency was added to the docker image, make sure it's also added to the native image and also documented in [here](#os-dependencies-for-each-supported-docker-image) readme, examples:
      - assuming "git" was added to the **[crypto](https://github.com/demisto/dockerfiles/tree/master/docker/crypto)** docker image, make sure it is also added to the native image and make sure its documented [here](#os-dependencies-for-each-supported-docker-image).
      - assuming "curl" was added to the **[readpdf](https://github.com/demisto/dockerfiles/tree/master/docker/readpdf)** docker image, make sure it is also added to the native image and make sure its documented [here](#os-dependencies-for-each-supported-docker-image).

Should be added to the optional section

Reply:
why? this is required

Reply:
The idea here is that we want the dev to know he needs to be able to run the integration on the native image. On his own he can understand that he needs the python/os dependencies to be sorted out. To elaborate and explain that is good, but not every developer needs to read it. Thats why I think it should go under a "read more about ... " type of section

Reply:
done